windows/src/developer/TIKE/compile/CompileKeymanWeb.pas

```diff
@@ -1697,6 +1709,24 @@ function TCompileKeymanWeb.ValidateLayoutFile(var sLayoutFile: string; const sVK
         end;
       end;
     end;
+
+    procedure TransformSpecialKeys14(var sLayoutFile: string);
+    var
+      i: Integer;
+    begin
+      // Rewrite Special key labels that are only supported in Keyman 14+
+      // This code is a little ugly but effective.
+      if not IsKeyboardVersion14OrLater then
+      begin
+        for i := 0 to High(CSpecialText14Map) do
+        begin
+          // Assumes the JSON output format will not change
+          if FDebug
+            then sLayoutFile := ReplaceStr(sLayoutFile, '"text": "'+CSpecialText14Map[i][0]+'"', '"text": this._v>13 ? "'+CSpecialText14Map[i][0]+'" : "'+CSpecialText14Map[i][1]+'"')
```

Comment:
If I'm not mistaken, this code would be run during construction of the `.KVKL` keyboard property - well, really, one of its sub-objects, right?  Would `this._v` then be referring to part of the `KVKL` object hierarchy under construction, or to the main keyboard-object's `_v` property?  `this`-scoping is well-known for being quite tricky in JS, so this feels like a possible weak point.

I'd like to see a freshly-compiled keyboard requiring use of the `_v` property tested within KMW to make sure the logic holds correctly.  While screenshots as proof would probably suffice, it'd be nice to have the actual compiled keyboard (used for those screenshots) for independent test use as well.

Reply:
> this code would be run during construction of the `.KVKL` keyboard property

Not quite. It's an assignment of an object literal, so has the scope of the containing object.

> I'd like to see a freshly-compiled keyboard requiring use of the `_v` property tested within KMW to make sure the logic holds correctly.

`test_touchlayout_14_3.kmn` can be compiled to demonstrate the solution. (I can't currently attach files to the issue due to a GH glitch, so here is a [zip](https://drive.google.com/file/d/10TiSrdl5jBjBVcXjLpNyy0l7Kdn873O5/view?usp=sharing) on Google Drive.)

I have manually tested within KeymanWeb and encountered no issues.